revolv/base/forms.py

```diff
@@ -21,6 +21,7 @@ def save(self, commit=True):
         last names.
         """
         user = super(SignupForm, self).save(commit=False)
+        user.save()
```

Comment:
something about the many to many relationships of the preferred_categories required that the user is saved before setting its attributes (I don't remember exactly)
would this work?
![screen shot 2015-06-05 at 2 28 40 pm](https://cloud.githubusercontent.com/assets/7706852/8015637/34d7702a-0b8f-11e5-953f-41e2f27fe6bb.png)